dashboard/app/controllers/user_levels_controller.rb

```diff
@@ -0,0 +1,30 @@
+class UserLevelsController < ApplicationController
```

Comment:
What is this class and how is it different from LevelsController, LevelSourcesController, or ScriptLevelsController? A class-level comment would be helpful here.